test/3x/torch/algorithms/w8a8/test_pt2e_w8a8.py

```diff
@@ -0,0 +1,65 @@
+import unittest
+from unittest.mock import patch
+
+import pytest
+import torch
+
+from neural_compressor.common.utils import logger
+from neural_compressor.torch.algorithms.w8a8_quant.core import W8A8StaticQuantizer
+
+
+class TestW8A8StaticQuantizer:
+
+    @staticmethod
+    def get_toy_model():
+        class Bar(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+
+            def forward(self, a: torch.Tensor, b: torch.Tensor) -> torch.Tensor:
+                x = a / (torch.abs(a) + 1)
+                if b.sum() < 0:
+                    b = b * -1
+                return x * b
+
+        inp1 = torch.randn(10)
+        inp2 = torch.randn(10)
+        example_inputs = (inp1, inp2)
+        bar = Bar()
+        return bar, example_inputs
+
+    def test_quantizer_on_llm(self):
+        from transformers import AutoModelForCausalLM, AutoTokenizer
+
+        model_name = "facebook/opt-125m"
+        model = AutoModelForCausalLM.from_pretrained(model_name)
+        tokenizer = AutoTokenizer.from_pretrained(model_name)
+        input_ids = tokenizer("Hello, my dog is cute", return_tensors="pt")["input_ids"]
+        example_inputs = (input_ids,)
+        quant_config = None
+        w8a8_static_quantizer = W8A8StaticQuantizer()
+        # prepare
+        prepare_model = w8a8_static_quantizer.prepare(model, quant_config, example_inputs=example_inputs)
+        # calibrate
+        for i in range(2):
+            prepare_model(*example_inputs)
+        # convert
+        converted_model = w8a8_static_quantizer.convert(prepare_model)
+        # inference
+        from torch._inductor import config
+
+        config.freezing = True
+        opt_model = torch.compile(converted_model)
```

Comment:
I think ipex and pt2e have different behavior here, which may confuse user.
BTW, does the opt_model here have save API?

Reply:
No, `torch.compile` is only required by pt2e quantization.